policyengine_us/variables/gov/states/or/tax/income/liheap/or_liheap_payout.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+class liheap_region(Variable):
+    value_type = str
+    entity = SPMUnit
+    definition_period = YEAR
+    defined_for = StateCode.OR
+    label = "LIHEAP Region"
+    reference = "https://www.oregon.gov/ohcs/energy-weatherization/Documents/2021-Energy-Assistance-Manual.pdf#page=55" 
+
+    def formula(SPMUnit, period, parameters):
+        county = SPMUnit("county_str", period)
+
+        region1_counties = parameters(period).gov.states["or"].liheap.region1_counties
+
+        region = select(
+            [county in region1_counties],  
+            ["Region 1"],                 
+            default="Region 2"            
+        )
+
+        return region
+
+class liheap_payout(Variable):
+    value_type = float
+    entity = SPMUnit
+    definition_period = YEAR
+    defined_for = StateCode.OR
+    label = "LIHEAP Payout"
+
+    def formula(SPMUnit, period, parameters):
+        payout = SPMUnit("utility_type", period)
```

Comment:
Make this a separate `.py` file 

```suggestion
class liheap_payout(Variable):
    value_type = float
    entity = SPMUnit
    definition_period = YEAR
    defined_for = StateCode.OR
    label = "LIHEAP Payout"

    def formula(spm_unit, period, parameters):
        utility_type = spm_unit("utility_type", period)
        unit_size = spm_unit("spm_unit_size", period)
        clipped_income_range = clip(unit_size, 1, 6)
        income_range = spm_unit("or_liheap_income_range", period)
        p = ......
        region_one = ...
        return where(p.region_one[unit_size][income_range][utility_type], p.region_two...)
        
```